src/test/java/net/sf/jabref/importer/fileformat/CopacImporterTest.java

```diff
@@ -76,17 +97,56 @@ public void testImportEntries() throws IOException {
     public void testImportEntries2() throws IOException {
         CopacImporter importer = new CopacImporter();
 
-        try (InputStream stream = CopacImporterTest.class
-                .getResourceAsStream("CopacImporterTest2.txt")) {
+        try (InputStream stream = CopacImporterTest.class.getResourceAsStream("CopacImporterTest2.txt")) {
             List<BibEntry> entries = importer.importEntries(stream, new OutputPrinterToNull());
             Assert.assertEquals(2, entries.size());
             BibEntry one = entries.get(0);
 
             Assert.assertEquals("Computing and operational research at the London Hospital", one.getField("title"));
+            Assert.assertEquals("Barber, Barry and Abbott, W.", one.getField("author"));
+            Assert.assertEquals("Computers in medicine series", one.getField("series"));
+            Assert.assertEquals("London :  Butterworths", one.getField("publisher"));
+            Assert.assertEquals("x, 102p, leaf : ill., form, port ; 22cm", one.getField("physicaldimensions"));
+            Assert.assertEquals("0407517006 (Pbk)", one.getField("isbn"));
+            Assert.assertEquals("Bibl.p.94-97. - Index", one.getField("note"));
+            Assert.assertEquals(
+                    "London Hospital and Medical College, Electronic data processing - Medicine, Computers - Hospital administration, Hospital planning, Operations research, Hospital equipment and supplies, Electronic data processing - Hospitals - Administration, Hospitals, London, London Hospital and Medical College, Records management, Applications of computer systems, to 1971",
+                    one.getField("keywords"));
+            Assert.assertEquals("TXT, PDF", one.getField("documenttype"));
+            Assert.assertEquals(
+                    "Aberdeen ; Birmingham ; Edinburgh ; Trinity College Dublin ; UCL (University College London)",
+                    one.getField("HL"));
```

Comment:
This implementation of CopacImporter knows the tags TI, AU, PY, PU, SE, IS, KW, NT, PD and DT. In this case Copac doesn't know what HL is. So to address this field you have to use the tag HL as field. This ressources already existed. In this case this is a test to watch whether the importer correctly handles with unknown tags.